diff --git a/debian/control b/debian/control
index 8223d84fc2..06877981d1 100644
--- a/debian/control
+++ b/debian/control
@@ -63,6 +63,10 @@ Build-Depends:
  libdc1394-dev [linux-any],
 # autodetected
  libdrm-dev [linux-any],
+# --enable-v4l2-request
+ linux-libc-dev [linux-any],
+# --enable-libudev
+ libudev-dev [linux-any],
 # autodected: ffnvcodec
  libffmpeg-nvenc-dev [amd64 arm64 i386],
 # --enable-libfontconfig
diff --git a/debian/rules b/debian/rules
index 24d252c4d2..837b86055c 100755
--- a/debian/rules
+++ b/debian/rules
@@ -118,6 +118,9 @@ endif
 # Some libraries are built only on linux.
 ifeq (linux,$(DEB_HOST_ARCH_OS))
 	CONFIG += --enable-libdc1394 \
+		--enable-v4l2_m2m \
+		--enable-libudev \
+		--enable-v4l2-request \
 		--enable-libdrm \
 		--enable-libiec61883
 endif
