diff --git a/debian/control b/debian/control
index 8223d84fc2..06877981d1 100644
--- a/debian/control
+++ b/debian/control
@@ -57,6 +57,10 @@ Build-Depends:
  libdc1394-dev [linux-any],
 # --enable-libdrm
  libdrm-dev [linux-any],
+# --enable-v4l2-request
+ linux-libc-dev [linux-any],
+# --enable-libudev
+ libudev-dev [linux-any],
 # autodected: ffnvcodec
  libffmpeg-nvenc-dev [amd64 arm64 i386],
 # --enable-libfontconfig
diff --git a/debian/rules b/debian/rules
index 24d252c4d2..837b86055c 100755
--- a/debian/rules
+++ b/debian/rules
@@ -144,6 +144,11 @@ endif
 # Some libraries are built only on linux.
 ifeq (linux,$(DEB_HOST_ARCH_OS))
 	CONFIG += --enable-libdc1394 \
+		--enable-v4l2_m2m \
+		--enable-libudev \
+		--enable-v4l2-request \
+		--disable-mmal \
+		--enable-sand \
 		--enable-libdrm \
 		--enable-libiec61883
 endif
--- a/tests/ref/fate/imgutils
+++ b/tests/ref/fate/imgutils
@@ -234,6 +234,9 @@
 nv42            planes: 2, linesizes:  64 128   0   0, plane_sizes:  3072  6144     0     0, plane_offsets:  3072     0     0, total_size: 9216
 y210be          planes: 1, linesizes: 256   0   0   0, plane_sizes: 12288     0     0     0, plane_offsets:     0     0     0, total_size: 12288
 y210le          planes: 1, linesizes: 256   0   0   0, plane_sizes: 12288     0     0     0, plane_offsets:     0     0     0, total_size: 12288
+sand128         planes: 2, linesizes:  64  64   0   0, plane_sizes:  3072  1536     0     0, plane_offsets:  3072     0     0, total_size: 4608
+sand64_10       planes: 2, linesizes: 128 128   0   0, plane_sizes:  6144  3072     0     0, plane_offsets:  6144     0     0, total_size: 9216
+sand64_16       planes: 2, linesizes: 128 128   0   0, plane_sizes:  6144  3072     0     0, plane_offsets:  6144     0     0, total_size: 9216
 x2rgb10le       planes: 1, linesizes: 256   0   0   0, plane_sizes: 12288     0     0     0, plane_offsets:     0     0     0, total_size: 12288
 x2rgb10be       planes: 1, linesizes: 256   0   0   0, plane_sizes: 12288     0     0     0, plane_offsets:     0     0     0, total_size: 12288
 x2bgr10le       planes: 1, linesizes: 256   0   0   0, plane_sizes: 12288     0     0     0, plane_offsets:     0     0     0, total_size: 12288
--- a/tests/ref/fate/sws-pixdesc-query
+++ b/tests/ref/fate/sws-pixdesc-query
@@ -21,6 +21,7 @@
   rgb48le
   rgba64be
   rgba64le
+  sand64_16
   ya16be
   ya16le
   yuv420p16be
@@ -65,6 +66,7 @@
   p210le
   p410be
   p410le
+  sand64_10
   x2bgr10be
   x2bgr10le
   x2rgb10be
@@ -213,6 +215,9 @@
   p410le
   p416be
   p416le
+  sand128
+  sand64_10
+  sand64_16
   uyvy422
   uyyvyy411
   xyz12be
@@ -737,6 +742,9 @@
   rgb8
   rgba64be
   rgba64le
+  sand128
+  sand64_10
+  sand64_16
   uyvy422
   uyyvyy411
   x2bgr10be
